spec/index.bs

```diff
@@ -1011,6 +1011,9 @@ but some [=valid=] objects may become [=invalid=].
 Objects are [=invalid=] from creation if it wasn't possible to create them.
 This can happen, for example, if the [=object descriptor=] doesn't describe a valid
 object, or if there is not enough memory to allocate a resource.
+It can also happen if another object needed to create a new object was invalid
```

Comment:
Had to read this sentence several times over because it sounded wrong. I think it's actually grammatically correct, it just felt really weird. Something like this reads more clearly to me. YMMV:

```suggestion
It can also happen if an object is created with or from another invalid object
(for example calling {{GPUTexture/createView()}} on an invalid {{GPUTexture}})
```

Reply:
I struggled with this, that's better, thanks.